### PR TITLE
Add methods get_awards_documents and get_qualification_documents

### PR DESCRIPTION
**Pull request by Krokop**
This change is [Reviewable](https://reviewable.io/reviews/openprocurement/openprocurement.client.python/63)

### CODE REVIEW

**Review by VDigitall — COMMENTED** [review]
Потрібно ще тести для цих методів.

**Comment by Krokop** [comment]
@VDigitall додав тести, потрібен ревью

File: openprocurement_client/tests/_server.py [hunk 1]
```diff
@@ -66,6 +66,20 @@ def tender_subpage(tender_id, subpage_name):
     subpage = tender_partition(tender_id, subpage_name)
     return dumps({"data": subpage})
 
+def tender_award_documets(tender_id, award_id):
```
Comment (VDigitall) [inline openprocurement_client/tests/_server.py]: `tender_award_documets` пропустили букву "n"

File: openprocurement_client/tests/_server.py [hunk 2]
```diff
@@ -272,6 +286,8 @@ def contract_partition(contract_id, part="contract"):
         "tender_patch": (TENDERS_PATH + "/<tender_id>", 'PATCH', tender_patch),
         "tender_document_create": (TENDERS_PATH + "/<tender_id>/documents", 'POST', tender_document_create),
         "tender_subpage": (TENDERS_PATH + "/<tender_id>/<subpage_name>", 'GET', tender_subpage),
+        "tender_award_documets": (TENDERS_PATH + "/<tender_id>/awards/<award_id>/documents", 'GET', tender_award_documets),
```
Comment (VDigitall) [inline openprocurement_client/tests/_server.py]: `tender_award_documets` пропустили букву "n"

**Review by VDigitall — COMMENTED** [review]
Виправіть в тестах tender_award_documets на tender_award_documents

**Review by VDigitall — APPROVED** [review]

**Comment by kroman0** [comment]
Reviewed 1 of 1 files at r1, 3 of 3 files at r2.
Review status: 1 of 3 files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/63)*

**Comment by kroman0** [comment]
Reviewed 2 of 2 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/63)*

**Comment by kroman0** [comment]
OK

---

Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/63#-:-KiEv12hXDE2-EIG_xVW:b32w0x)*

**Comment by VDigitall** [comment]
Reviewed 1 of 1 files at r1, 3 of 3 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/63)*

**Pull request merged**